src/components/menu-list/menu-list.stories.mdx

```diff
@@ -1,179 +1,173 @@
-import { useState } from 'react';
-import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
-import { MenuList, MenuListItem } from '.'
+import { useState } from "react";
+import { Meta, Story, Preview, Props } from "@storybook/addon-docs/blocks";
+import { MenuList, MenuListItem } from ".";
 
-<Meta title="MenuList" parameters={{ info: { disable: true }}} />
+<Meta title="MenuList" parameters={{ info: { disable: true } }} />
 
 # MenuList
+
 A set of related items that can be shown or hidden.
 
 Children can be shown or hidden by clicking their title. Useful to create simple accordion or menu patterns.
 
+<Preview>
+  <div
+    style={{
+      backgroundColor: "red",
+      textAlign: "center",
+      color: "white",
+      padding: 20,
+      fontWeight: "bold",
+      marginBottom: 10,
+    }}
+  >
+    Create has been deprecated please see
```

Comment:
This should reference MenuList not Create

Reply:
Ah, forgot to update and copying and pasting in!